dev-meetings/2025/2025-06-20/README.md

```diff
@@ -21,6 +25,67 @@
+   * Removing the setup in every tutorial [#5816](https://github.com/gammapy/gammapy/issues/5816)
+
+### Features
+* How to support the default plot labels [#5894](https://github.com/gammapy/gammapy/pull/5894)
+* How to correctly test the stacking functionality for LightCurveEstimator [#5898](https://github.com/gammapy/gammapy/pull/5898)
+
+# Minutes
+## SAT IKC latest news
+* Bruno presented news about the CTAO Science Analysis Tools project. 
+* Bruno reminded the meeting that the In kind contribution includes two year of bug fixes after the hand-over of the tools.
+* Formally there are 38 FTEs in the project, but by Bruno's count there are currently around 42 FTEs already used up and experience tells us that the work needed is frequently underestimated.
+* Bruno showed a release plan  
+
+    * SAT release 0 to happen by September 30 in 2025
+
+    * SAT release 1 to happen by January 31 in 2026
+
+    * Final acceptance release to be delivered by end of September 2027
+
+* Bruno suggests that the release plan should be aligned with the IKC review process 
+* Bruno suggests we start working on defining a project dedicated to the maintenance and upgrade of the SAT
+* Bruno warns that there is a coming cost in terms of work on the verification documentation.
+* Bruno provides an overview of work defining the VOD format
+
+### Questions
+* Quentin asked who would provide for the science data challenge, would the Gammapy devs have to provide this? Bruno says this is a good question, not clear how it will work but CTAO  will somehow provide support.
+* After a question from Quentin, Daniel clarifies that the SAT will mostly be an import of gammapy that lives in and is managed via gitlab with additional CTAO-specific documentation and possibly some additional functionalities. The plan is being worked out, discussed and will be presented to CTAO Computing. They have to approve it.
+* Atreyee asks if this includes if also the documentation should be hosted on gitlab, Daniel answers yes. 
+* Atreyee asks what happens with user questions. Daniel answers that it is not clear at the moment. Bruno remarks that this is a good example of an issue that would be resolved by an IKC agreement covering support and future upgrades.
+* In a discussion following a question Atreyee it is highlighted that there is a risk that the Gammapy version accepted, and required to be supported for 2 years, could end up out of step with the Gammapy LTS releases, requiring the support of two separate LTS releases! Another reason to create a IKC agreement covering support.
+* Atreyee asked a further question I was not able to record
+
+## VERITAS tutorial
+* Samantha presents her draft tutorial showing VERITAS users how to use gammapy to make a real analysis
+* Samantha found she needs to use a local start catalogue file, because queries to Simbad tended to crash; this filed would be hosted in gammapy-data.
```

Comment:
```suggestion
* Samantha found she needs to use a local star catalogue file, because queries to Simbad tended to crash; this filed would be hosted in gammapy-data.
```